ava.config.cjs

```diff
@@ -0,0 +1,10 @@
+module.exports = () => {
+  return {
+    require: ["./test/helpers/ava.setup.js"],
+    ignoredByWatcher: ["!**/*.{js,vue}"],
+    babel: true,
```

Comment:
As a personal preference, I find it useful to keep the files prop in there, even though it adds verbosity. I find it makes it much easier to test specific features in watch mode (but others have their preferences too). I've tried it both ways (the way you have here, and with files list; I can work faster with "test.only" and only specific files enabled)